schema/mysql/v57/cadence/versioned/v0.4/blob_size.sql

```diff
@@ -12,7 +12,7 @@ ALTER TABLE timer_info_maps MODIFY COLUMN data MEDIUMBLOB;
 ALTER TABLE child_execution_info_maps MODIFY COLUMN data MEDIUMBLOB;
 ALTER TABLE request_cancel_info_maps MODIFY COLUMN data MEDIUMBLOB;
 ALTER TABLE signal_info_maps MODIFY COLUMN data MEDIUMBLOB;
-ALTER TABLE buffered_replication_task_maps MODIFY COLUMN data MEDIUMBLOB;
+ALTER TABLE buffered_replication_task_maps MODIFY COLUMN new_run_history MEDIUMBLOB;
```

Comment:
Why are we renaming this column?

Reply:
This is the copy paste error from Temporal. Our column named this